### PR TITLE
Added a C# promises implementation to the list of implementations in other languages.

### PR DESCRIPTION
**Pull request by ashleydavis**
My company has built a promises implementation and I'd like to have it added to the list of implementations in other languages.

### CODE REVIEW

**Comment by briancavalier** [comment]
Pretty cool, @ashleydavis.  LGTM.  Any concerns, @domenic?

File: implementations.md [hunk 1]
```diff
@@ -304,5 +304,10 @@ We've been excited to discover that the popularity of Promises/A+ has led to a n
             <td><a href="https://github.com/stevan/promises-perl">stevan/promises-perl</a></td>
             <td>An implementation of the Promises/A+ pattern for asynchronous programming in Perl.</td>
         </tr>
+        <tr>
+            <td>C#</td>
+            <td><a href="https://github.com/Real-Serious-Games/c-sharp-promise">Real-Serious-Games/c-sharp-promise</a></td>
+            <td>An unit-tested implementation of the Promises/A+ pattern for asynchronous programming in C#.</td>
```
Comment (domenic) [inline implementations.md]: s/An/A/

**Comment by domenic** [comment]
Awesome stuff! One nitpick on the commit but happy to have this.

**Comment by briancavalier** [comment]
Sorry for the delay, @ashleydavis.  Merged via 67707dcf3d43ba7be56af1670f80cc49bbad69b1

**Comment by ashleydavis** [comment]
@briancavalier Thank you!

**Pull request closed without merging**